### DISCUSSION TITLE
mata baaaaaaaaaaa

### DISCUSSION THREAD

**Discussion by dinumifonseka** [dinumifonseka/Hospital-Web-#5] [Announcements]
chaka baka

**Comment by DhanujaRehan** [comment]
hrima agaei buru gani

**Reply by dinumifonseka** [reply]
ai halow

**Reply by DhanujaRehan** [reply]
balanna ... homepage ekata hdapu ekama ekata connect krala hriyanne naha auth ekak hdanna one

**Reply by DhanujaRehan** [reply]
eka thamai connect krala thiyenne oya balanna gihilla

**Comment by DhanujaRehan** [comment]
balahanko ara aluthin hadapu ekata connect krala thiyenne mona css file ekakda kiyala

**Reply by dinumifonseka** [reply]
kalin css ekama tha mn oyagen ahuva ne
tho ha kiuve moda miniha

**Reply by DhanujaRehan** [reply]
![image](https://github.com/user-attachments/assets/4666b0d9-2402-4f03-af38-aa0624cb0c45)
balanna ekamai thiyenne css ekak aluthin dala naha

**Reply by dinumifonseka** [reply]
damme na thamai

**Reply by DhanujaRehan** [reply]
> kalin css ekama tha mn oyagen ahuva ne tho ha kiuve moda miniha

manussayo html, header, nav bar, root kiyana css tika mn kiwwe ee widihatama danna kiyala

**Reply by DhanujaRehan** [reply]
> damme na thamai

aluth ekak hdanna

**Comment by DhanujaRehan** [comment]
![logo](https://github.com/user-attachments/assets/7d64270d-28e5-4154-94dd-e10c6cd324bc)
meka download krnna
